src/main/java/com/speaker/service/validator/message/MessageDataValidatorImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.speaker.service.validator.message;
+
+import com.speaker.dto.MessageDTO;
+import com.speaker.dto.ValidatorError;
+import com.speaker.entities.EntityField;
+import com.speaker.service.validator.AbstractValidator;
+import com.speaker.service.validator.Validator;
+import com.speaker.service.validator.type.ErrorType;
+import com.speaker.service.validator.type.FieldType;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+import static java.util.Objects.isNull;
+
+@Component
+public class MessageDataValidatorImpl extends AbstractValidator implements Validator<MessageDTO> {
+    private final long CURRENT_TIME = System.currentTimeMillis();
```

Comment:
this is not the current time!
in this case `CURRENT_TIME` - the time where the bean was created

Reply:
Comments accepted, corrections made. The time is changed
    private final long CURRENT_TIME = LocalDateTime.now().atZone(ZoneId.systemDefault()).toInstant().toEpochMilli();

Reply:
I mean that it final variable, and fill one time when created bean in spring context. If the progect was started two days ago this variable will have date = `current time - 2 days`. 
Please do not extract this into a final variable, use it in the method.